### PR TITLE
chore(pfAppLauncher): launcher icon moved to the right to prevent cutoff

### LINKED ISSUE #595: App launcher update
The angular version of the app launcher menu is getting cut off here: http://www.patternfly.org/pattern-library/application-framework/launcher/#/code/angular

This relates to: https://github.com/patternfly/patternfly-org/issues/452

### PR DESCRIPTION
**Pull request by amarie401**
## Description
App launcher update to move launcher icon to the right in order to prevent cutoff on pf-org. This would close #595 

screen shot 2017-08-31 at 5 18 00 pm

https://rawgit.com/amarie401/angular-patternfly/fix-app-launcher-menu-dist/dist/docs/index.html#/api/patternfly.navigation.component:pfApplicationLauncher

## PR Checklist

- [X] Unit tests are included
- [X] Screenshots are attached (if there are visual changes in the UI)
- [ ] A Designer (@beanh66) is assigned as a reviewer (if there are visual changes in the UI)
- [ ] A CSS rep (@cshinn) is assigned as a reviewer (if there are visual changes in the UI)

@jeff-phillips-18 @dtaylor113 @LHinson @cdcabrera

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
This is a doc only change, could you make this a chore rather than a fix?

**Comment by jeff-phillips-18** [comment]
There are no visual changes here (except for the example) so this doesn't require the designer or css rep reviews.

**Comment by amarie401** [comment]
@jeff-phillips-18 chore? Do you mean adding these changes also to the actual component?

File: misc/ng-docs.css [hunk 1]
```diff
@@ -44,6 +44,12 @@
   margin-right: 0px;
 }
 
+a.navbar-brand img {
+  height: inherit;
+  width: inherit;
+  margin-top: inherit;
+}
+
```
Comment (jeff-phillips-18) [inline misc/ng-docs.css]: Wondering if this has any adverse effects on other examples. Did you look at the vertical navigation example to verify it does not?

Comment (amarie401) [reply misc/ng-docs.css]: I checked vertical nav and it looks to be fine.

**Comment by amarie401** [comment]
I am going to attempt to add config examples to show the list/grid and icons/no icons variations. Please do not merge yet.

**Comment by amarie401** [comment]
Have added icons/no icons for list view

**Pull request merged**